setup.py

```diff
@@ -81,6 +81,7 @@ def get_version():
         'botocore>=1.19.37',
         'boto3>=1.17.58',
         'ipython>=7.16.1',
+        'neo4j',
```

Comment:
Do we/should we pin the version here?

Reply:
Yes; pinned to latest version, since we know that is currently working